frontend/packages/yki/src/components/registration/PublicIdentificationGrid.tsx

```diff
@@ -98,79 +227,17 @@ export const PublicIdentificationGrid = () => {
-                    size="large"
-                    variant={Variant.Contained}
-                    color={Color.Secondary}
-                    onClick={() => {
-                      // TODO: init authentication for suomi.fi
-                      dispatch(increaseActiveStep());
-                      navigate(
-                        AppRoutes.ExamSessionRegistration.replace(
-                          /:examSessionId$/,
-                          `${examSession.id}`
-                        )
-                      );
-                    }}
-                    data-testid="public-registration__identify-button"
-                    disabled={false}
-                  >
-                    {t('steps.identify.suomiFiButtonText')}
-                  </CustomButton>
                   <Text>
-                    <Trans t={t} i18nKey={'steps.identify.withoutFinnishSSN'} />
+                    <li>{t('steps.identify.instructions.time')}</li>
                   </Text>
-                  <div className="columns gapped align-items-start">
-                    <CustomTextField
-                      className="public-registration__grid__form-container__registration-text-field"
-                      error={showCustomTextFieldError('email')}
-                      placeholder={t('steps.identify.emailPlacehodler')}
-                      variant={Variant.Outlined}
-                      type={TextFieldTypes.Email}
-                      value={email}
-                      onChange={handleChange('email')}
-                      onBlur={handleErrors('email')}
-                      helperText={fieldErrors['email']}
-                    ></CustomTextField>
-                    <CustomButton
-                      size="large"
-                      className="public-registration__grid__form-container__registration-button"
-                      variant={Variant.Contained}
-                      color={Color.Secondary}
-                      onClick={() => {
-                        dispatch(setIsEmailRegistration(true));
-                        dispatch(increaseActiveStep());
-                        navigate(
-                          AppRoutes.ExamSessionRegistration.replace(
-                            /:examSessionId$/,
-                            `${examSession.id}`
-                          )
-                        );
-                      }}
-                      data-testid="public-registration__identify-button"
-                      disabled={false}
-                    >
-                      {t('steps.identify.emailButtonText')}
-                    </CustomButton>
-                  </div>
+                </ul>
+                <div className="gapped rows">
+                  <IdentificationSelection />
                   <div className="columns margin-top-lg justify-content-center">
                     <PublicRegistrationControlButtons
```

Comment:
Ei tarvii ehkä tässä yhteydessä tehdä tälle vielä mitään, mutta Peruuta ilmoittautuminen -painike vois olla varmaan piilotettuna sen jälkeen kun sähköpostilinkin on tilannut

Reply:
Joo, pitäis varmaan piilottaa. Taidan myös nimetä sen uudelleen jossain weeklyssäkin pyöritellyn mukaisesti -> `Keskeytä ilmoittautuminen`.